model/task/cedar_service.go

```diff
@@ -1,4 +1,4 @@
-package testresult
+package task
```

Comment:
Is there a reason the service implementations can't stay in `model/testresult`?

Reply:
It would create a cyclic dependency error of imports, e.g. `model/task` -> `model/testresult` -> `model/task`